docs/Installation/InstallationOnAppleSilicon.md

```diff
@@ -64,31 +57,31 @@ pip install jupyterlab
 
 ### 3. Install dependencies with Homebrew
 
-Most of spectre's dependencies beyond python can be installed using the
-[homebrew](https://brew.sh) package manager. First, if you haven't
-already, install Homebrew by
-following the instructions on the [homebrew](https://brew.sh) homepage. Then,
-run the following to install a fortran compiler and other dependencies:
+Most of spectre's dependencies can be installed using the
+[Homebrew](https://brew.sh) package manager. First, if you haven't already,
+install Homebrew by following the instructions on the
+[Homebrew](https://brew.sh) homepage. Then, run the following to install a
+Fortran compiler and other dependencies:
+
 ```
-brew install gcc
-brew install boost gsl cmake doxygen catch2 openblas
-brew install ccache autoconf automake jemalloc hdf5 pybind11 yaml-cpp
+brew install gcc autoconf automake ccache cmake
+brew install boost catch2 doxygen gsl hdf5 jemalloc openblas yaml-cpp
```

Comment:
Why no pybind11 here? Is this taken care of by step 2 above (i.e., the requirements or dev_requirements)?

Reply:
yes, that's one of the earlier commits in this PR.